### PR TITLE
CORE-1819 Add Sprint Submission details to the Review page

### PR DESCRIPTION
**Pull request by mellogarrett**
## Description

[CORE-1819]

- Add a `SprintSubmissionInfoCard` to the side of the page once a TL selects one to review

sprint-submission-info-card-demo

## Type of Change 

- New feature (non-breaking change which adds functionality)

[CORE-1819]: https://lambdaschool.atlassian.net/browse/CORE-1819

### CODE REVIEW

File: pages/no-config-sprint-review.tsx [hunk 1]
```diff
@@ -237,5 +239,32 @@ function verifySprintSubmissionIdOptions(sprintSubmissionIdOptions: any[]): NoCo
...
+  if (technicalSubmission)
+    fields.push({
+      label: 'Sprint Submission URL(s):',
+      value: technicalSubmission.toString(),
+    });
+  if (selfRating)
+    fields.push({
+      label: 'Self-Evaluation:',
+      value: selfRating.toString(),
+    });
```
Comment (MarkDKoslow) [inline pages/no-config-sprint-review.tsx:259]: Is the projection returning these fields?

File: pages/no-config-sprint-review.tsx [hunk 2]
```diff
@@ -237,5 +239,32 @@ function verifySprintSubmissionIdOptions(sprintSubmissionIdOptions: any[]): NoCo
   return sprintSubmissionIdOptions;
 }
 
+function SprintSubmissionInfo() {
+  const [form] = useForm();
+  const sprintSubmission = form.getQuestion('sprintSubmissionId') as NoConfigDropdownQuestion;
+  const selectedChoice = sprintSubmission.choices.find((choice) => choice.value === sprintSubmission.answer);
+  if (!selectedChoice) return <div data-testid="hidden-element" />;
```
Comment (MarkDKoslow) [inline pages/no-config-sprint-review.tsx:246]: clever 👏

File: pages/no-config-sprint-review.tsx (hunk 1 above)
Comment (mellogarrett) [reply pages/no-config-sprint-review.tsx:259]: It is as of `atlas-db@31.8.0`

File: pages/no-config-sprint-review.tsx [hunk 3]
```diff
@@ -39,6 +40,7 @@ const NoConfigSprintReview = ({ projectManagerIdOptions, sprintSubmissionIdOptio
         key: 'sprintSubmissionId',
       }}
     />
+    <SprintSubmissionInfo />
```
Comment (MarkDKoslow) [inline pages/no-config-sprint-review.tsx:43]: Perks of no-config in action 👏 👏 👏

**Pull request merged**